com/course/zenkouv/src/main/java/lecture6/gitHub/GitHub.java

```diff
@@ -0,0 +1,34 @@
+package lecture6.gitHub;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lecture6.IRequestExecutor;
+import lecture6.constant.GitHubApiConstants;
+import lecture6.constant.RequestType;
+import lecture6.gitHub.controller.GitHubController;
+import lecture6.model.GitHubUser;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+public class GitHub {
+
+    private GitHub() {
+    }
+
+    public static void sendCollaboratorInvite(String name, IRequestExecutor requestExecutor) throws IOException, URISyntaxException {
```

Comment:
use object instead of static methods
put IRequestExecutor in constructor 
create instance of GitHubController in this constructor

Reply:
fix